src/common/clioptions/cli_test.go

```diff
@@ -44,6 +45,7 @@ func TestCliArgParsing(t *testing.T) {
 			Output:         "cli",
 			OutputJSONFile: "",
 			TagGroups:      []string{"git", "code2cloud"},
+			DryRun:         true,
```

Comment:
@hi-artem Let's test what happens when the `DryRun` option isn't supplied - we expect it to not crash with a value of `false`, right?

Reply:
@nimrodkor Addressed your comment.

Should I also be concerned about some of the CI checks failing?